packages/monaco/src/index.tsx

```diff
@@ -40,7 +40,7 @@ export function CodePanel(props: CodePanelProps) {
   const isPlayground = pathname.includes('playground');
   const { toast } = useToast();
   const [tsErrors, setTsErrors] = useState<TsErrors>();
-  const [isTestPanelExpanded, setIsTestPanelExpanded] = useState(false);
+  const [isTestPanelExpanded, setIsTestPanelExpanded] = useState(!isPlayground);
```

Comment:
can just use `true`

Reply:
I thought that it shouldn't be opened by default in the playground view, but if that is not the case, then sure, I'll change it to true.